src/main/java/com/stripe/net/RequestMetrics.java

```diff
@@ -0,0 +1,21 @@
+package com.stripe.net;
+
+public class RequestMetrics {
+    public String request_id;
+    public long request_duration_ms;
```

Comment:
does GSON Support annotations for json field aliases? It would be better to not mix non standard java variable names just for json support. https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html

Reply:
Yes! Updated.